lib/PaginationEmbed.js

```diff
@@ -10,20 +14,20 @@ const ReactionHandler = require('eris-reactions');
 class PaginationEmbed {
     /**
      * Constructor for the Embed Paginator
-     * @param {Eris#Message} message A message object emitted from a messageCreate event coming from Eris, used as an invoker. 
-     * @param {Object[]} pages An array containing all embed objects 
-     * @param {Object} options An optional options object for overwriting defaults
-     * @property {Boolean} options.showPageNumbers Whether or not to show the current page index over the embed. Defaults to: true
-     * @property {Boolean} options.extendedButtons Whether or not to show extended control buttons besides standard pagination (First & Last page, deleting)
-     * @property {Boolean} options.cycling Cycle through all embeds jumping from the first page to the last page on going back and from the last page to the first page going forth. Defaults to: false
-     * @property {Number} options.maxMatches How often the reaction handler should listen for a reaction (How often the paginator can be used). Defaults to: 50. Maximum: 100
-     * @property {Number} options.timeout How long the paginator should work before the reaction listener times out. Defaults to: 300000ms (5 minutes). Maximum: 900000ms (15 minutes)
-     * @property {String} options.deleteButton Emoji which should be used as the delete button. This MUST be a unicode emoji! Defaults to: 🗑
-     * @property {String} options.firstButton Emoji which should be used as the first page button. This MUST be a unicode emoji! Defaults to: ⏮
-     * @property {String} options.lastButton Emoji which should be used as the last page button. This MUST be a unicode emoji! Defaults to: ⏭
-     * @property {String} options.backButton Emoji which should be used as the back button. This MUST be a unicode emoji! Defaults to: ⬅
-     * @property {String} options.forthButton Emoji which should be used as the forth button. This MUST be a unicode emoji! Defaults to: ➡
-     * @property {Number} options.startPage Which page of the submitted embed array shoulb be shown first. Defaults to: 1 (The 1st page / element in the array)
+     * @param {Message} message A message object emitted from a messageCreate event coming from Eris, used as an invoker. 
+     * @param {EmbedBase[]} pages An array containing all embed objects 
+     * @param {Object} [options] An optional options object for overwriting defaults
+     * @param {Boolean} [options.showPageNumbers] Whether or not to show the current page index over the embed. Defaults to: true

```

Comment:
changing these property tags to param actually does nothing and is semantically incorrect. this would look better as a typedef anyway

Reply:
property tags don't show JSDoc for it though? 🤔

Reply:
I don't know if this project has a special parsing for jsdoc, but standard jsdoc reads param and property about the same. again a typedef will be more accurate (to match the interface from the typings)